tests/MongoDB.Driver.Legacy.VB.Tests/MongoDB.Driver.Legacy.VB.Tests.vbproj

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="packages\Microsoft.NET.Test.Sdk.15.8.0\build\net45\Microsoft.Net.Test.Sdk.props" Condition="Exists('packages\Microsoft.NET.Test.Sdk.15.8.0\build\net45\Microsoft.Net.Test.Sdk.props')" />
+  <Import Project="packages\Microsoft.CodeCoverage.15.8.0\build\netstandard1.0\Microsoft.CodeCoverage.props" Condition="Exists('packages\Microsoft.CodeCoverage.15.8.0\build\netstandard1.0\Microsoft.CodeCoverage.props')" />
+  <Import Project="packages\xunit.core.2.4.0\build\xunit.core.props" Condition="Exists('packages\xunit.core.2.4.0\build\xunit.core.props')" />
```

Comment:
I don't know how to review this commit.

Did you make these changes manually or using the GUI?

Reply:
Sorry for the "drip" comments: I made this change using the GUI, updating the packages that we had updated in the .csproj files.